code/game/machinery/vending_types.dm

```diff
@@ -346,24 +346,24 @@
 		/obj/item/storage/fancy/cigar = 5
 	)
 	prices = list(
-		/obj/item/storage/fancy/cigarettes/rugged = 200,
-		/obj/item/storage/fancy/cigarettes = 245,
-		/obj/item/storage/fancy/cigarettes/dromedaryco = 270,
-		/obj/item/storage/fancy/cigarettes/nicotine = 325,
-		/obj/item/storage/fancy/cigarettes/pra = 230,
-		/obj/item/storage/chewables/rollable/bad = 40,
-		/obj/item/storage/chewables/rollable = 50,
-		/obj/item/storage/chewables/rollable/fine = 60,
-		/obj/item/storage/chewables/rollable/nico = 90,
-		/obj/item/storage/chewables/tobacco/bad = 60,
-		/obj/item/storage/chewables/tobacco = 90,
-		/obj/item/storage/chewables/tobacco/fine = 120,
-		/obj/item/storage/fancy/chewables/tobacco/nico = 150,
+		/obj/item/storage/fancy/cigarettes/rugged = 10,
+		/obj/item/storage/fancy/cigarettes = 15,
+		/obj/item/storage/fancy/cigarettes/dromedaryco = 25,
+		/obj/item/storage/fancy/cigarettes/nicotine = 30,
+		/obj/item/storage/fancy/cigarettes/pra = 25,
+		/obj/item/storage/chewables/rollable/bad = 5,
+		/obj/item/storage/chewables/rollable = 10,
+		/obj/item/storage/chewables/rollable/fine = 15,
+		/obj/item/storage/chewables/rollable/nico = 20,
+		/obj/item/storage/chewables/tobacco/bad = 20,
+		/obj/item/storage/chewables/tobacco = 25,
+		/obj/item/storage/chewables/tobacco/fine = 40,
+		/obj/item/storage/fancy/chewables/tobacco/nico = 30,
 		/obj/item/storage/box/matches = 12,
 		/obj/item/flame/lighter/random = 12,
-		/obj/item/storage/cigfilters = 20,
-		/obj/item/storage/fancy/cigpaper = 32,
-		/obj/item/storage/fancy/cigpaper/fine = 48,
+		/obj/item/storage/cigfilters = 10,
+		/obj/item/storage/fancy/cigpaper = 10,
+		/obj/item/storage/fancy/cigpaper/fine = 25,
```

Comment:
They were already cheap to begin with. Again, recently added (**BY ME.**) , so no reason for this change, because they weren't affected by the 2 years arc you keep talking about.

Reply:
Not when you have to buy three of them, and these prices were basically balanced around Jackboot's shit.

Reply:
...Why do you need to buy three of them? They're a box. Of more than one paper.

Reply:
You need to buy one of each.

Reply:
Yeah, and each contains more than one filter, more than one tobacco leaf, more than one cigarette paper. And you don't even need the cigarette filters.